src/Robo/Commands/Artifact/AcHooksCommand.php

```diff
@@ -71,6 +71,30 @@ public function postCodeUpdate($site, $target_env, $source_branch, $deployed_tag
     }
   }
 
+  /**
+   * Execute sql-sanitize against a database hosted in AC Cloud.
+   *
+   * This is intended to be called from db-scrub.sh cloud hook.
+   *
+   * @param string $site
+   *   The site name. E.g., site1.
+   * @param string $target_env
+   *   The cloud env. E.g., dev
+   *
+   * @command artifact:ac-hooks:db-scrub
+   *
+   * @throws \Exception
+   */
+  public function dbScrub($site, $target_env) {
```

Comment:
I'd prefer to pass all available arguments to the command, `$site, $target_env, $source_branch, $deployed_tag, $repo_url, $repo_type`. This would allow BLT to make use of these in the future without requiring a new major version or re-generation of hooks.

Reply:
@grasmash i can update, but the db-scrub has these variables (not exactly the same as you mention above):

```
site="$1"
target_env="$2"
db_name="$3"
source_env="$4"
```

Reply:
thoughts?